Add rev share section (#40)
Sovereign-Labs/sovereign-book: src/8-0-complying-with-revenue-share.md

# Revenue Share for Premium Components

> When using Sovereign SDK's premium components (such as the ultra-low latency soft-confirming sequencer), applications that generate revenue must comply with the Sovereign Permissionless Commercial License. This guide walks you through the implementation.

Author: ```suggestion
When using Sovereign SDK's premium components (such as the ultra-low latency soft-confirming sequencer), applications that generate revenue must comply with the Sovereign Permissionless Commercial License. This guide walks you through the implementation.
**Note:** Non-commercial applications and development are all exempt from the revenue share requirements.
```

# Revenue Share for Premium Components

When using Sovereign SDK's premium components (such as the ultra-low latency soft-confirming sequencer), applications that generate revenue must comply with the Sovereign Permissionless Commercial License. This guide walks you through the implementation.

> **Note:** Non-commercial applications and development are all exempt from the revenue share requirements.

Reviewer: ```suggestion
**Note:** Non-commercial and non-production use of the SDK is exempt from the revenue share requirements.
```

^ to align more closely with the license terminology
